packages/client/src/comp/states/BaseRegistration.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { RegistrationStatus } from '../../common'
+import styled from 'styled-components'
+
+const I = styled.i`
+  color: var(--primary);
+  &:hover { cursor: help; }
+`
+
+const Red = styled.span`
+  color: var(--danger);
+`
+
+export type BaseRegistrationStatus = RegistrationStatus | 'Error' | 'Ignored' | 'Loading'
+
+interface Props {
+  registrationStatus: BaseRegistrationStatus
+  setIsOpen: (_: boolean) => void
+}
+
+export const BaseRegistration: React.FC<Props> = ({ registrationStatus, setIsOpen }) => {
+  if (registrationStatus === null) return null
+
+  if (registrationStatus === 'Loading') return <>
+    <b>Voter Registration:</b> Loading <i className="fa fa-spinner fa-spin"/>
+  </>
+  if (registrationStatus === 'Active') return <>
+    <b>Voter Registration:</b> Active <i style={{ color: '#4caf50' }} className="fa fa-check"/>
+  </>
+  if (registrationStatus === 'Ignored') return <>
+    <b>Voter Registration:</b>{' '}
+    Ignored{' '}
+    <I className="fa fa-question-circle" onClick={() => setIsOpen(true)}/>
+  </>
+  return <>
+    <b>Voter Registration:</b>{' '}
+    <Red>{registrationStatus === 'Error' ? 'Error' : 'Invalid'}</Red>{' '}
```

Comment:
Let's display the actual status and only display 'Invalid' if it's null (or would it be more appropriate to display 'Error' if it returns null.

Reply:
We can show the actual status, but `registrationStatus` will never be null on `BaseRegistration.tsx` since `Base.tsx` assigns default values for null responses on `client.isRegistered`, is that okay for this case? 

https://github.com/vote-by-mail/website/blob/31a0116e5daf7b84c3ea557ba1d16465e8f87479/packages/client/src/comp/states/Base.tsx#L110-L124